Documentation/quickstart/index.rst

```diff
@@ -1,16 +1,18 @@
 .. todo::
-  This should be a brief introduction to how NuttX works and how to get it running. A widely used board could be targeted to create a brief demonstration on how to clone, configure, build, flash and run NuttX.
-  For now, the content that is here presented is what remains of the old Porting Guide which is not already in another section of this documentation.
 
 ==========
 Quickstart
 ==========
 
-The present Quickstart guide is a work in progress and only contains some basic information. In the meantime you can look at NuttX main `README <https://github.com/apache/incubator-nuttx/blob/master/README.md>`_ file which contains some information that will help you get started.
-
 .. toctree::
   :maxdepth: 1
   
+  quickstart.rst
+  install.rst
+  compiling.rst
+  running.rst
+  configuring.rst
+  debugging.rst
   organization.rst
```

Comment:
These two files are leftovers of the previous docs which ended up here. As you already cover this information somewhat I would try to get it is important from these files, move it to the ones you added and then remove them.

Reply:
@v01d Thanks for noting this. I saw that too, wasn't sure what to do with them. I will think about it and edit the these files as you suggest.